packages/components/src/components/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { getAttrsAsJson } from "../lib/get-attrs-as-json";
+import React, { useState, useEffect, Suspense, lazy } from "react";
+import { createPortal } from "react-dom";
+import { CourierSdk } from "./CourierSdk";
+
+const Toast = lazy(() => import("./Toast"));
+const Inbox = lazy(() => import("./Inbox"));
+
+export const CourierComponents: React.FunctionComponent = () => {
+  const componentConfigs = window.courierConfig?.components;
+  const initialInbox = document.querySelector("courier-inbox");
+  const [inboxElement, setInboxElement] = useState(initialInbox ?? undefined);
+
+  const inboxConfig = {
+    ...componentConfigs?.inbox,
+    ...getAttrsAsJson(inboxElement),
+  };
+
+  const initialToast = document.querySelector("courier-toast");
+  const [toastElement, setToastElement] = useState(initialToast ?? undefined);
+
+  const toastConfig = {
+    ...componentConfigs?.toast,
+    ...getAttrsAsJson(toastElement),
+  };
+
+  useEffect(() => {
+    const observer = new MutationObserver((mutationsList) => {
```

Comment:
can we create a function with a name for this to understand what this is supposed to be doing?